### PR TITLE
[DOCFIX] Add a doc for Trino

### PR DESCRIPTION
**Pull request by yuzhu**
Add a page for Trino DB.

### CODE REVIEW

**Comment by yuzhu** [comment]
It will initially look very similar to PrestoDb's doc, but expect it to deviate later.

File: docs/en/compute/Trino.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+Alluxio allows Trino to access data regardless of the data source and transparently cache frequently
+accessed data (e.g., tables commonly used) into Alluxio distributed storage.
+Co-locating Alluxio workers with Trino workers improves data locality and reduces the I/O access
+latency when other storage systems are remote or the network is slow or congested.
+
+* Table of Contents
+{:toc}
+
+## Using Trino with the Alluxio Catalog Service
+Currently, there are 2 ways to enable Trino to interact with Alluxio:
```
Comment (LuQQiu) [inline docs/en/compute/Trino.md:23]: Add a line before

**Review by LuQQiu — APPROVED** [review]
LGTM, left one minor comment

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please

**Pull request merged**